.github/workflows/release-container.yml

```diff
@@ -4,6 +4,8 @@
     push:
       tags:
         - '[0-9]+.[0-9]+.[0-9]+'
+    schedule:
+      - cron: '0 0 * * *'
```

Comment:
It's [recommended](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) not to run crons at the top of an hour: "To decrease the chance of delay, schedule your workflow to run at a different time of the hour"

So something like `'7 0 * * *'` might work more reliably

Reply:
ok, done!

Reply:
You might want to consider something more frequent than nightly, perhaps hourly. This would be possible without a lot of resource consumption, using the more efficient check as I suggested

Reply:
This was not a requirement in my understanding.
From other discussions, I got the impression that we want to re-spin images a maximum of once a day.

I'm happy to re-consider your proposal if this assumption has changed! cc. @abstractj @stianst

Reply:
The reason I suggest a more frequent check is to more quickly respond to new versions of UBI, and limit the potential time any CVEs go unfixed. Ideally this would be event driven, but I don't know of any API to enable that. Checking more frequently doesn't necessarily mean more than one image is made per day